salt/modules/virt.py

```diff
@@ -2809,6 +3006,184 @@ def _get_with_unit(node):
+            "del": salt.utils.xmlutil.del_attribute("mode"),
+        },
+        {
+            "path": "cpu:check",
+            "xpath": "cpu",
+            "get": lambda n: n.get("check"),
+            "set": lambda n, v: n.set("check", str(v)),
+            "del": salt.utils.xmlutil.del_attribute("check"),
+        },
+        {"path": "cpu:model:name", "xpath": "cpu/model"},
+        {
+            "path": "cpu:model:fallback",
+            "xpath": "cpu/model",
+            "get": lambda n: n.get("fallback"),
+            "set": lambda n, v: n.set("fallback", str(v)),
+            "del": salt.utils.xmlutil.del_attribute("fallback"),
+        },
+        {
+            "path": "cpu:model:vendor_id",
+            "xpath": "cpu/model",
+            "get": lambda n: n.get("vendor_id"),
+            "set": lambda n, v: n.set("vendor_id", str(v)),
+            "del": salt.utils.xmlutil.del_attribute("vendor_id"),
+        },
+        {
+            "path": "cpu:vendor",
+            "xpath": "cpu/vendor",
+            "del": salt.utils.xmlutil.del_text,
+        },
+        {
+            "path": "cpu:topology:sockets",
+            "xpath": "cpu/topology",
+            "get": lambda n: n.get("sockets"),
+            "set": lambda n, v: n.set("sockets", str(v)),
+            "del": salt.utils.xmlutil.del_attribute("sockets"),
+        },
+        {
+            "path": "cpu:topology:cores",
+            "xpath": "cpu/topology",
+            "get": lambda n: n.get("cores"),
+            "set": lambda n, v: n.set("cores", str(v)),
+            "del": salt.utils.xmlutil.del_attribute("cores"),
+        },
+        {
+            "path": "cpu:topology:threads",
+            "xpath": "cpu/topology",
+            "get": lambda n: n.get("threads"),
+            "set": lambda n, v: n.set("threads", str(v)),
+            "del": salt.utils.xmlutil.del_attribute("threads"),
+        },
+        {
+            "path": "cpu:cache:level",
+            "xpath": "cpu/cache",
+            "get": lambda n: n.get("level"),
+            "set": lambda n, v: n.set("level", str(v)),
+            "del": salt.utils.xmlutil.del_attribute("level"),
+        },
+        {
+            "path": "cpu:cache:mode",
+            "xpath": "cpu/cache",
+            "get": lambda n: n.get("mode"),
+            "set": lambda n, v: n.set("mode", str(v)),
+            "del": salt.utils.xmlutil.del_attribute("mode"),
+        },
+        {
+            "path": "cpu:feature:name",
+            "xpath": "cpu/feature",
+            "get": lambda n: n.get("name"),
+            "set": lambda n, v: n.set("name", str(v)),
+            "del": salt.utils.xmlutil.del_attribute("name"),
+        },
+        {
+            "path": "cpu:feature:policy",
+            "xpath": "cpu/feature",
+            "get": lambda n: n.get("policy"),
+            "set": lambda n, v: n.set("policy", str(v)),
+            "del": salt.utils.xmlutil.del_attribute("policy"),
+        },
```

Comment:
There is an issue here: there can (are likely to) be several features. We need to handle things like:
```
cpu:
  features:
    vmx: require
    lahf_lm: disable
```